package.json

```diff
@@ -8,12 +8,13 @@
     "metalsmith-collections": "^0.7.0",
     "metalsmith-concat": "^2.0.0",
     "metalsmith-markdown": "~0.2.1",
+    "metalsmith-metadata": "~0.0.1",
     "metalsmith-permalinks": "~0.4.0",
     "metalsmith-sass": "~1.2.1",
-    "metalsmith-swig-helpers": "^1.2.1",
+    "metalsmith-swig-helpers": "git://github.com/madeofpeople/metalsmith-swig-helpers.git#609094c03bf0d0df534a39f2f66c9ac07a08e1bf",
```

Comment:
@madeofpeople I think 1.2.1 doesn't have the dynamic filter. May need a bump to 1.2.2?